### DISCUSSION TITLE
Permission to use someone's page

### DISCUSSION THREAD

**Discussion by Sahad76** [community/community#34472] [New to GitHub] [answered]
Hi, does anyone know who to contact to use the page below,

https://maxzilla60.github.io/Spelling-with-Elements/ 

We essentially wanted to use this for our social media competition using the periodic table.

Thanks,
Sahad

**Comment by janhartje** [comment, marked as answer]
I would guess https://github.com/Maxzilla60, at least it's his Repository.
